rust/hyperlane-base/src/db/typed_db.rs

```diff
@@ -1,17 +1,15 @@
-use derive_new::new;
-
-use hyperlane_core::{Decode, Encode};
+use hyperlane_core::{Decode, Encode, HyperlaneDomain};
 
 use crate::db::{DbError, DB};
 
 type Result<T> = std::result::Result<T, DbError>;
 
 /// DB handle for storing data tied to a specific type/entity.
 ///
-/// Key structure: ```<type_prefix>_<additional_prefix(es)>_<key>```
```

Comment:
What was the previous "type_prefix", and why is it okay to remove?

Reply:
It was poorly named, it was only being used as the domain name. This is not a change to logic, just clarity.